webpack.config.js

```diff
@@ -0,0 +1,11 @@
+const path = require('path');
```

Comment:
Please feel free to suggest / write a better Webpack config, I'm still new to it.